requirements/environment.yml

```diff
@@ -13,6 +13,6 @@ dependencies:
   - statsmodels
   - jinja2
   - pip:
-    - neo>=0.10.0
+    - neo>=0.13.0
```

Comment:
Could you please also update this in `requirements/requirements.txt`?
Thank you :)

Reply:
done